cert/selfsigned_test.go

```diff
@@ -15,6 +17,12 @@ const (
 var (
 	extraIPs     = []string{"1.1.1.1", "123.123.123.1", "199.189.12.12"}
 	extraDomains = []string{"home", "and", "away"}
+	privKeyBytes = [32]byte{
```

Comment:
This is still not used anywhere. I wonder why the linter doesn't catch that. Maybe we disabled the "unused" linter for tests...